src/librustdoc/html/format.rs

```diff
@@ -900,16 +892,7 @@ impl clean::Type {
 }
 
 impl clean::Impl {
-    crate fn print<'a>(&'a self, cache: &'a Cache) -> impl fmt::Display + 'a {
-        self.print_inner(true, false, cache)
-    }
-
-    fn print_inner<'a>(
-        &'a self,
-        link_trait: bool,
-        use_absolute: bool,
-        cache: &'a Cache,
-    ) -> impl fmt::Display + 'a {
+    crate fn print<'a>(&'a self, cache: &'a Cache, use_absolute: bool) -> impl fmt::Display + 'a {
```

Comment:
Is this ever called with `use_absolute: true`? If not, can you remove the parameter?

Reply:
It's used here: https://github.com/rust-lang/rust/blob/1daddb47d2516b503941e8893193789f030b4971/src/librustdoc/html/render/mod.rs#L3773

Where AFAICT `use_absolute` can be `true` from https://github.com/rust-lang/rust/blob/1daddb47d2516b503941e8893193789f030b4971/src/librustdoc/html/render/mod.rs#L2476-L2483